Makefile

```diff
@@ -0,0 +1,4 @@
+
+.PHONY: run_server
+run_server:
+	@uvicorn how2meet.main:app --reload --log-level debug --port 8000
```

Comment:
Where does this come from/where does it get called? I'm not super familiar with makefiles, and have only kinda used them for compiling C for microcontrollers?